containers/rust-postgres/README.md

```diff
@@ -0,0 +1,78 @@
+# Rust & PostgreSQL
+
+## Summary
+
+*Develop applications with Rust and PostgreSQL. Includes a Rust application container and PostgreSQL server.*
+
+| Metadata | Value |
+|----------|-------|
+| *Contributors* | The VS Code Team |
+| *Categories* | Core, Languages |
+| *Definition type* | Docker Compose |
+| *Published images* | mcr.microsoft.com/vscode/devcontainers/rust |
+| *Available image variants* | buster, bullseye ([full list](https://mcr.microsoft.com/v2/vscode/devcontainers/rust/tags/list)) |
+| *Published image architecture(s)* | x86-64, arm64/aarch64 for `bullseye` variant |
+| *Works in Codespaces* | Yes |
+| *Container host OS support* | Linux, macOS, Windows |
+| *Container OS* | Debian |
+| *Languages, platforms* | Rust |
+
+## Using this definition
+
+This definition creates two containers, one for Rust and one for PostgreSQL. VS Code will attach to the Rust dev container, and from within that container the PostgreSQL container will be available on **`localhost`** port 5432. The default database is named `postgres` with a user of `postgres` whose password is `postgres`, and if desired this may be changed in `docker-compose.yml`. Data is stored in a volume named `postgres-data`.
+
+While the definition itself works unmodified, you can select the version of Debian the container uses by updating the `VARIANT` arg in `.devcontainer/docker-compose.yml` (and rebuilding if you've already created the container).
+
+```yaml
+build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        # Use the VARIANT arg to pick a Debian OS version: buster, bullseye
+        # Use bullseye when on local on arm64/Apple Silicon.
+        VARIANT: buster
+```
+
+Beyond `git`, this image / `Dockerfile` includes `zsh`, [Oh My Zsh!](https://ohmyz.sh/), a non-root `vscode` user with `sudo` access, and a set of common dependencies for development.
```

Comment:
Is Zsh a necessary dependency or a nice to have? I would say that we should remove unnecessary dependencies like Zsh and simply give instructions on how they can be added back in.